lib/components/viewers/next-arrival-for-pattern.tsx

```diff
@@ -64,11 +74,21 @@ function NextArrivalForPattern(props: Props) {
   const title = `${routeName} ${toHeadsign}`
 
   return (
-    <div className="next-arrival-row">
+    <div
+      className="next-arrival-row"
+      style={{
+        backgroundColor: routeColor,
+        color: getMostReadableTextColor(routeColor, route?.textColor)
```

Comment:
I am seeing inconsistent colors with this stop: http://localhost:9966/#/stop/5:21097, where the text for route 140 can be both black and white on the same screen.

Reply:
This is because the related stops endpoint includes `textColor` while the stops endpoint doesn't. Repairing this involves a lot of redux plumbing, but it should be fixed once we source all the stop info from one endpoint in the future.